app/models/publications/monthly_statistics/statistics_data_processor.rb

```diff
@@ -0,0 +1,32 @@
+module Publications
+  module MonthlyStatistics
+    class StatisticsDataProcessor
```

Comment:
Shouldn't this have some specs? Seems the classes in this folder aren't unit tested.

Reply:
Done